share/picongpu/examples/FoilLCT/include/picongpu/param/particle.param

```diff
@@ -104,9 +92,10 @@ namespace picongpu
                  *
                  * unit: none
                  */
-                static constexpr uint32_t numParticlesPerCell = 1u;
+                static constexpr uint32_t numParticlesPerCell = TYPICAL_PARTICLES_PER_CELL;
```

Comment:
please check that this is not a silent setup change (in all changed files), maybe it should be one. 
It is not required to be `TYPICAL_PARTICLES_PER_CELL`

Reply:
good catch, fixed!